members/taksh/data.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "takshch",
```

Comment:
This id is supposed to be your RDS id - **taksh** as you've mentioned in the mapping.json file.

Reply:
Changed to "taksh"